playbooks/robusta_playbooks/pod_enrichments.py

```diff
@@ -81,5 +82,5 @@ def pod_node_graph_enricher(pod_event: PodEvent, params: ResourceGraphEnricherPa
     if not node:
         logging.warning(f"Node {pod.spec.nodeName} not found for pod {pod.metadata.name}")
         return
-    graph_enrichment = create_node_graph_enrichment(params, node)
+    graph_enrichment = create_node_graph_enrichment(params, node, metrics_legends_labels=["pod", "node"])
```

Comment:
This is a node graph, I ion't think the pod is relevant here
I also think it has an `instance` label, and not a `node`
Maybe for Node graphs, we should put `instance` and `node`
But you need to verify the UI knows to handle a missing label